core-services/prow/02_config/_plugins.yaml

```diff
@@ -677,6 +677,78 @@ bugzilla:
             release-4.8:
               target_release: OCS 4.8.0
               validate_by_default: false
+        cluster-logging-operator:
+          branches:
+            '*':
+              add_external_link: true
+              allowed_groups:
+              - redhat
+              - qe_staff
+              exclude_defaults: true
+              is_open: true
+              state_after_close:
+                status: NEW
+              state_after_merge:
+                status: MODIFIED
+              state_after_validation:
+                status: POST
+              valid_states:
+              - status: NEW
+              - status: ASSIGNED
+              - status: ON_DEV
+            release-4.6:
+              dependent_bug_target_releases:
```

Comment:
pretty sure the whole idea here is you do NOT want to require the bug to depend on another bug, when merging into the release-4.6 branch.  So i would assume you don't need this stanza at all.

Reply:
@bparees the other examples I have seen they target the release, for example, to be 4.7 https://github.com/openshift/release/pull/17128/files#diff-a9035bbaee5af66130c81cfd0b97df589a7d2e22e87ad9d3800851407967c07bR626-L629

Reply:
that link goes here